scripts/leapfrog/playbooks/elasticsearch-postleap-pip-upgrade.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Upgrade pip on Elasticsearch container
+  hosts: elasticsearch_all
+  user: root
+  roles:
+    - role: "pip_install"
```

Comment:
Why do we need the pip_install role run within an existing elk container and why does this need to be done post-leap? shouldn't our elk role take care of this already?

Reply:
The fact is the way to upgrade elasticsearch require us keeping old elasticsearch container. Well the old container won't match pip config which elasticsearch upgrade playbook asked. The most straightforward solution is simply running pip role on elasticsreach container.

Reply:
The fact is the way to upgrade elasticsearch requires us keeping old elasticsearch container. Well the old container won't match pip config which elasticsearch upgrade playbook asked. The most straightforward solution is simply running pip role on elasticsreach container.